server/aws-lsp-json/src/language-server/jsonServer.ts

```diff
@@ -133,17 +138,14 @@ export const JsonServerFactory =
         }
     }
 
-async function getSchema(url: string) {
-    const response = await fetch(url)
-    const schema = await (await response.blob()).text()
-
-    return schema
+export function createCustomJsonLanguageServer(customService: JsonLanguageService) {
+    return JsonServerFactory(new MutuallyExclusiveLanguageService([customService]))
 }
 
 export const CreateJsonLanguageServer = (
     defaultSchemaUri: string,
-    allowComments: boolean = true,
-    uriResolver: (url: string) => Promise<string> = getSchema
+    allowComments?: boolean,
```

Comment:
Is it a miss or intentional that we're changing the default value for `allowComments`? It used to be set to `true` in this line when it wasn't defined, but now it stays `undefined` in this function and when it reaches [jsonLanguageService.ts#L31](https://github.com/aws/language-servers/blob/dd76df1d5c6f7741c34494b516ba66224a78de34/server/aws-lsp-json/src/language-service/jsonLanguageService.ts#L31) it will become `false` instead.

Technically it's a breaking change too / not backwards compatible, so I recommend we avoid it

Reply:
Sure, thanks for the callout - I shall update it. 
And yes, there will be a followup to make this changes in Yaml too.